src/actions/fetchInstitution.js

```diff
@@ -17,8 +18,15 @@ export default function fetchInstitution(institution, fetchFilings = true) {
             throw new Error(json && `${json.status}: ${json.statusText}`)
           }
           dispatch(receiveInstitution(json))
-          if (json.filings && fetchFilings) {
+          if (json.filings.length !== 0 && fetchFilings) {
```

Comment:
The response here returns `filings: []` so we can see the `length !== 0` and fetch the current filing OR create the new one with the current `lei` and `filingPeriod`.